modules/virt-importing-vm-datavolume.adoc

```diff
@@ -75,7 +97,12 @@ spec:
         storageClassName: local
       source:
         http:
+ifdef::virt-vm-image[]
           url: https://download.fedoraproject.org/pub/fedora/linux/releases/28/Cloud/x86_64/images/Fedora-Cloud-Base-28-1.1.x86_64.qcow2 <1>
+endif::virt-vm-image[]
+ifdef::virt-container-disk[]
+          registry: docker://registry.fedoraproject.org/fedora <1>
```

Comment:
Can we get an example here that works if some attempts to copy and paste it? 

For example, ```"docker://kubevirt/fedora-cloud-container-disk-demo:latest"``` is one of our upstream images that would actually work on their cluster.

Reply:
registry.fedoraproject.org/fedora works for me as an image but if the kubevirt-maintained image is a better source I can update the example
I can see that I've messed the conditionals here though.

Reply:
the issue is that ```registry.fedoraproject.org/fedora works``` doesn't have a disk image in it. It's a generic container image. We're actually placing a qcow disk in the ```kubevirt/fedora-cloud-container-disk-demo:latest``` image

Reply:
Ahh, understood. Thanks for clarifying.